polygeist/test/polygeist-opt/sycl/cast.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: polygeist-opt --convert-polygeist-to-llvm="use-bare-ptr-memref-call-conv" --split-input-file %s | FileCheck %s
```

Comment:
Can you add also a test to ensure that a input memref that has an explicit address space (e.g. memref<1xi64,4>) is converted to a llvm.ptr in the same address space pls.

Reply:
Done!